react/week1/todo-app/src/App.js

```diff
@@ -0,0 +1,14 @@
+import Todolist from './todos';
+import './App.css';
+
+function App() {
+  return (
```

Comment:
maybe work a bit with the formatting here. for example

    return <div className="todo-app">{<Todolist />}</div>;

is a little easier to read